Gifski/util.swift

```diff
@@ -2449,7 +2449,6 @@ extension NSViewController {
 		}, completionHandler: {
```

Comment:
Unrelated, but why are we not using https://developer.apple.com/documentation/appkit/nsviewcontroller/1434431-present? We could use it with a custom animator: https://stackoverflow.com/a/28493373/64949

Reply:
No particular reason, found that content switching API and used it.